tools/clang/test/CodeGenHLSL/quick-test/matrix_orientation_transpose_in_function.hlsl

```diff
@@ -0,0 +1,16 @@
+// RUN: %dxc /T vs_6_0 /E main > %s | FileCheck %s | XFail
+
+// Regression test for a bug where the transpose isn't performed,
```

Comment:
better to mention bug id for easy reference later on.

Reply:
Addressed in latest commit.